app/src/main/java/org/oppia/android/app/home/topiclist/TopicSummaryViewModel.kt

```diff
@@ -29,8 +21,6 @@ class TopicSummaryViewModel(
   @ColorInt
```

Comment:
I think I can remove background color from here too.

Reply:
I think you're right--I removed this locally in an in-progress PR. Please go ahead and remove it here. Seems like a nice opportunistic fix.

Reply:
Done.